cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/StreamAdminModules.java

```diff
@@ -81,7 +116,7 @@ protected void configure() {
         bind(StreamMetaStore.class).to(MDSStreamMetaStore.class).in(Singleton.class);
         bind(StreamConsumerStateStoreFactory.class).to(HBaseStreamConsumerStateStoreFactory.class).in(Singleton.class);
         bind(StreamAdmin.class).to(FileStreamAdmin.class).in(Singleton.class);
-        bind(StreamConsumerFactory.class).to(HBaseStreamFileConsumerFactory.class).in(Singleton.class);
+        bind(StreamConsumerFactory.class).to(streamConsumerFactory).in(Singleton.class);
```

Comment:
Have fixed for StreamAdminModules; assume this comment also applies to SystemDatasetRuntimeModule.